lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +244,94 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.article = Article(full_url, article_id)
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        text_div = article_soup.find('div', {'class': 'field-type-text-with-summary'})
+        self.article.text = '\n'.join(text for paragraph in text_div.find_all('p')
+                                      if (text := paragraph.text.strip()))
+        if not self.article.text:
```

Comment:
it is unnecessary to throw an error. code that handles it has access to text and can decide what to do with it

Reply:
Fixed